### PR TITLE
0.14.0 release

### PR DESCRIPTION
**Pull request by pirapira**
Making a release now to provide the new interface:  [`Removed checks in MonitoringService.canMonitor() until the design is determined.`](https://github.com/raiden-network/raiden-contracts/compare/master...pirapira:0.14.0?expand=1#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12)

### CODE REVIEW

**Comment by pirapira** [comment]
Codecov complains because no CI tests touch `setup.py`.  I think `setup.py` should not be counted  https://github.com/raiden-network/raiden-contracts/issues/724

File: raiden_contracts/data/gas.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
     "EndpointRegistry.registerEndpoint": 48408,
     "MonitoringService.claimReward": 42679,
-    "MonitoringService.monitor": 203858,
-    "OneToN.claim": 90914,
+    "MonitoringService.monitor": 201313,
+    "OneToN.claim": 90897,
```
Comment (palango) [inline raiden_contracts/data/gas.json]: I remember that there was an issue with raiden when these were lowered, but when switching to the new contracts completely this shouldn't be a problem.

File: raiden_contracts/data_0.10.0/gas.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "EndpointRegistry.registerEndpoint": 48408,
+    "MonitoringService.claimReward": 42679,
+    "MonitoringService.monitor": 201313,
+    "OneToN.claim": 90897,
+    "SecretRegistry.registerSecret": 45757,
+    "TokenNetwork DEPLOYMENT": 4031160,
+    "TokenNetwork.closeChannel": 111226,
```
Comment (palango) [inline raiden_contracts/data_0.10.0/gas.json]: Ah, my bad above. We have a file per version now.

File: raiden_contracts/data/gas.json (hunk 1 above)
Comment (pirapira) [reply raiden_contracts/data/gas.json]: When the last problem happened, the numbers lived in a global `raiden_contracts/constants.py`.  That set of numbers affected all versions. Now the situation should be better. `gas.json` lives in `data_x.y.z` so the numbers are local to a particular version.

But I have to try the new way to see if any problem happens.

File: raiden_contracts/data_0.10.0/gas.json (hunk 2 above)
Comment (pirapira) [reply raiden_contracts/data_0.10.0/gas.json]: Ah, I replied before seeing this.

Comment (pirapira) [reply raiden_contracts/data_0.10.0/gas.json]: One disadvantage of "everything per version" is a big release PR like this one.

**Pull request merged**